zyte_common_items/base.py

```diff
@@ -36,12 +42,17 @@ class _ItemBase:
     __slots__ = ("_unknown_fields_dict",)
 
 
-@export
-@attr.define(slots=True)
+@attrs.define(slots=True)
 class Item(_ItemBase):
     def __attrs_post_init__(self):
         self._unknown_fields_dict = {}
 
+    def asdict(self):
+        adapter = ItemAdapter(self)
+        result = adapter.asdict()
+        _remove_empty_values(result)
+        return result
```

Comment:
In autoextract-poet there was a custom adapter (https://github.com/scrapinghub/autoextract-poet/blob/master/autoextract_poet/adapters.py); I think it was used to preserve unknown fields. 

Questions:

1. Do we need to copy the approach, or are unknown fields handled in some other way? 
2. If we have a custom adapter, would it make sense to have _remove_empty_values somewhere in the adapter? I'm not sure it's possible to override `adapter.asdict()` properly, it might require changes to itemadapter.